docs/tutorial_samples/12_ommerGoodFiller.yml

```diff
@@ -0,0 +1,73 @@
+12_ommerGood:
+  genesisBlockHeader:
+    bloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'
+    coinbase: 0x1111111111111111111111111111111111111111
+    difficulty: '131072'
+    extraData: '0x42'
+    gasUsed: '0'
+    mixHash: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421'
+    nonce: '0x0102030405060708'
+    number: '0'
+    parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000'
+    receiptTrie: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421'
+    stateRoot: '0xf99eb1626cfa6db435c0836235942d7ccaa935f1ae247d3f1c21e495685f903a'
+    timestamp: '0x03b6'
+    transactionsTrie: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421'
+    gasLimit: 10240000
+    baseFeePerGas: 1000
+  sealEngine: NoProof
+
+
+  pre:
+    # Externally owned account (the result of entropy being all zeros)
+    0x9858EfFD232B4033E47d90003D41EC34EcaEda94:
+      balance: 0x10000000000
+      code: ''
+      nonce: 1
+      storage: {}
+
+
+
+  blocks:
+  - blocknumber: 1
+    transactions: []
+
+  - blocknumber: 2
+    transactions: []
+
+  - blocknumber: 3
+    transactions: []
+
+  - blocknumber: 4
+    transactions: []
+    blockHeader:
+      stateRoot: 0x73c5e0631444838bfc2d8368cc4c11a15596de9b586606ebdfe8ba03c45f1039
```

Comment:
this is a state root we need to overwrite because now the state has a mining reward for the uncles 
since we don't have a protocol that asks a client to generate a block with uncles. 
we need to correct this on retesteth side as retesteth constructing the block with uncles but it can't calculate the post state hash